File: festival-react/src/models/applicant.ts
import type { Competence } from "./competence"
import type { Region } from "./region"
import type { User } from "./user"

export type Applicant = {
    id: number,

    userId: number,
    user: User,
    
    competenceId: number,
    competence: Competence

    regionId: number,
    region: Region
}

File: festival-api/Models/Applicant.cs
namespace festival_api.Models;

public class Applicant
{
    public int Id {get; set;}

    public int UserId {get; set;}
    public User? User {get; set;}

    public int RegionId {get; set;}
    public Region? Region {get; set;}
    
    public int CompetenceId {get; set;}
    public Competence? Competence {get; set;}
}